packages/ace-linters/src/message-types.ts

```diff
@@ -91,7 +92,7 @@ export class ChangeMessage extends BaseMessage {
 }
 
 export class DeltasMessage extends BaseMessage {
-    type: MessageType = MessageType.applyDelta;
+    type: MessageType.applyDelta = MessageType.applyDelta;
```

Comment:
Why is this not inferred automatically?

Reply:
Not sure, seems limitation on TS side, without it there is no narrowing in switch-case blocks